package.json

```diff
@@ -11,15 +11,21 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "bower": "^1.7.9",
     "grunt": "0.4.5",
+    "grunt-cli": "^1.2.0",
     "grunt-contrib-connect": "0.5.0",
     "grunt-contrib-cssmin": "0.7.0",
     "grunt-contrib-watch": "0.5.3",
     "grunt-sass": "1.1.0",
     "grunt-swig-templates": "0.1.4",
     "grunt-version": "0.3.3",
     "jit-grunt": "0.9.1",
-    "node-bourbon": "4.2.3",
-    "load-grunt-tasks": "0.2.0"
+    "load-grunt-tasks": "0.2.0",
+    "node-bourbon": "4.2.3"
+  },
+  "scripts": {
+    "test": "grunt build",
+    "postinstall": "bower install"
```

Comment:
We don't usually put `bower install` in `package.json` Might be worth putting it in `install` step in `travis.yml` unless @henrahmagix suggested otherwise?